examples/index.html

```diff
@@ -38,15 +47,20 @@
 <pre id="internal"></pre>
 </body>
 <script>
+const mac = CodeMirror.keyMap.default === CodeMirror.keyMap.macDefault;
+const modifier = mac ? "Cmd" : "Ctrl";
+
 var pyretCM = CodeMirror(document.getElementById("editor"), {
   extraKeys: {
     "Tab": "indentAuto",
     "Esc Left": "goBackwardSexp",
     "Alt-Left": "goBackwardSexp",
     "Esc Right": "goForwardSexp",
     "Alt-Right": "goForwardSexp",
-    "Ctrl-Left": "goBackwardToken",
-    "Ctrl-Right": "goForwardToken"
+    [`${modifier}-Left`]: "goBackwardToken",
+    [`${modifier}-Right`]: "goForwardToken",
```

Comment:
At first I thought "One tricky thing here is that if anyone relied on Ctrl-Left/Right on Mac, they'd be confused now"

But those are for workspace switching! I think this is probably a strict improvement, even if not related to the PR.

Reply:
This is just in the demo code editor as well. I realized I rely on Cmd-Left/Right to jump to the start and end of lines and so these aren't actually reflected within CPO itself.